acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/validatorslashing/MultiPeersStandAloneVcBlocksAcceptanceTest.java

```diff
@@ -39,43 +38,25 @@ public class MultiPeersStandAloneVcBlocksAcceptanceTest
       shouldShutDownWhenOwnedValidatorSlashed_StandAloneVC_MultiplePeers_SlashingThroughBlock_NoSlashingEventsGossip(
           final SlashingEventType slashingEventType) throws Exception {
 
-    final int genesisTime = timeProvider.getTimeInSeconds().plus(10).intValue();
-    final UInt64 altairEpoch = UInt64.valueOf(100);
+    final int genesisTime = timeProvider.getTimeInSeconds().plus(30).intValue();
 
     final TekuBeaconNode firstTekuNode =
         createTekuBeaconNode(
             TekuNodeConfigBuilder.createBeaconNode()
                 .withGenesisTime(genesisTime)
                 .withNetwork(network)
                 .withRealNetwork()
-                .withAltairEpoch(altairEpoch)
+                .withSubscribeAllSubnetsEnabled()
                 .withInteropValidators(0, 32)
                 .build());
 
-    firstTekuNode.start();
-
-    firstTekuNode.waitForEpochAtOrAbove(2);
-
-    final int slashedValidatorIndex = 34;
-    final BLSKeyPair slashedValidatorKeyPair = getBlsKeyPair(slashedValidatorIndex);
-    final int slotInThirdEpoch =
-        firstTekuNode.getSpec().forMilestone(SpecMilestone.ALTAIR).getSlotsPerEpoch() * 2 + 3;
-
-    postSlashing(
-        firstTekuNode,
-        UInt64.valueOf(slotInThirdEpoch),
-        UInt64.valueOf(slashedValidatorIndex),
-        slashedValidatorKeyPair.getSecretKey(),
-        slashingEventType);
-
     final TekuBeaconNode secondBeaconNode =
         createTekuBeaconNode(
             TekuNodeConfigBuilder.createBeaconNode()
                 .withGenesisTime(genesisTime)
                 .withNetwork(network)
                 .withRealNetwork()
-                .withRealNetwork()
-                .withAltairEpoch(altairEpoch)
```

Comment:
is removal of `withAltairEpoch` intentional?

Reply:
think so, although this was added by Paul's commit, we never actually reached the epoch where Altair was set.

Reply:
yes it was pointless to have altair at epoch 100 when we're finishing in a few epochs...